lib/ddtrace/ext/profiling.rb

```diff
@@ -4,6 +4,13 @@ module Profiling
       ENV_MAX_FRAMES = 'DD_PROFILING_MAX_FRAMES'.freeze
       ENV_MAX_TIME_USAGE_PCT = 'DD_PROFILING_MAX_TIME_USAGE_PCT'.freeze
       ENV_IGNORE_PROFILER = 'DD_PROFILING_IGNORE_PROFILER'.freeze
+
+      module Pprof
```

Comment:
A fair suggestion; I might do this later, but not really inclined to do this at the moment because of how many updates I'd have to make. Maybe in another PR later.